.github/workflows/cron.yml

```diff
@@ -43,7 +43,9 @@ jobs:
       with:
         dotnet-version: ${{matrix.dotnetversion}}
     - name: Install Pulumi CLI
-      uses: pulumi/actions@v4
+      uses: pulumi/actions@5ffa6249eba838c2a624d5315a28fb8c7b3dabb7
```

Comment:
Do we really want this pinned? Isn't latest v4 always going to be fine for us?

Reply:
It's pinned just to get the latest changes from `pulumi/actions`.  We should probably do a release there so the latest v4 includes the changes to allow us to install the dev version.